catkin_tools/commands/catkin.py

```diff
@@ -18,7 +18,10 @@
 from datetime import date
 from shlex import quote as cmd_quote
 
-import pkg_resources
+try:
+    from importlib.metadata import distribution, entry_points
+except ImportError:
+    from importlib_metadata import distribution, entry_points
```

Comment:
importlib.metadata is always available because catkin_tools only supports 3.7+ so this branch is never hit

Reply:
This is only available from 3.8,https://docs.python.org/3/library/importlib.metadata.html

Reply:
I mean 3.8+ https://github.com/catkin/catkin_tools/blob/main/.github/workflows/workflow.yml#L12